test/e2e/functional_test.go

```diff
@@ -189,6 +189,8 @@ func (s *FunctionalSuite) TestMetricsWithWebhook() {
 		Contains("argo_events_event_processing_duration_milliseconds").
 		Contains("argo_events_events_processing_failed_total")
 
+	time.Sleep(3 * time.Second)
```

Comment:
Do we need this?

Reply:
i had some weird behavior with the kafka e2e test, and for some reason it helped, but i can remove it.